backend/canvas_app_explorer/lti1p3.py

```diff
@@ -12,19 +12,29 @@
 from django.urls import reverse
 from django.views.decorators.csrf import csrf_exempt
 from django.views.decorators.http import require_POST
-from pylti1p3.contrib.django import DjangoOIDCLogin, DjangoMessageLaunch, \
-    DjangoCacheDataStorage, DjangoDbToolConf
+from pylti1p3.contrib.django import (
+    DjangoCacheDataStorage, DjangoDbToolConf, DjangoMessageLaunch, DjangoOIDCLogin
+)
+from pylti1p3.exception import LtiException
 
 from .canvas_roles import STAFF_COURSE_ROLES
 
+
 logger = logging.getLogger(__name__)
 
+COURSE_MEMBERSHIP = 'http://purl.imsglobal.org/vocab/lis/v2/membership'
```

Comment:
Nothing is changed for the COURSE_MEMBERSHIP value

Reply:
Nevermind CacheConfig is a class-like object so it's fine named like this.